Makefile

```diff
@@ -447,7 +447,7 @@ CMAKE_SYSTEM_NAME := Windows
 endif
 
 CONFIGURE_FLAGS += --host=$(XHOST_TRIPLE) CC=$(CC_PATH) CXX=$(CXX_PATH)
-CMAKE_FLAGS += -DCMAKE_C_COMPILER=$(CC_PATH) -DCMAKE_CXX_COMPILER=$(CXX_PATH) -DCMAKE_SYSTEM_NAME=$(CMAKE_SYSTEM_NAME)
+XCMAKE_FLAGS := $(CMAKE_FLAGS) -DCMAKE_C_COMPILER=$(CC_PATH) -DCMAKE_CXX_COMPILER=$(CXX_PATH) -DCMAKE_SYSTEM_NAME=$(CMAKE_SYSTEM_NAME)
```

Comment:
Is `XCMAKE_FLAGS` standard? If not, consider adding a comment explaining that the variable contains cross-compilation-specific flags.

Reply:
Not even a little bit standard, but then again none of this stuff is. Good point, though: a comment is always appreciated. Done!